### PR TITLE
Rename advanced section

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@scheib @jyasskin @g-ortuno Is "Intermediate" better?

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
gentle ping @jyasskin

**Comment by addyosmani** [comment]
I think intermediate makes sense if you have a good sense for what would now get classified as 'advanced' :) Either way, those cc'd should have a good view on this. LGTM otherwise.

File: web-bluetooth/index.html [hunk 1]
```diff
@@ -17,6 +17,6 @@
 <p><a href="device-disconnect.html">Device Disconnect</a> - disconnect and get notified from a disconnection of a BLE Device after connecting to it.</p>
 <p><a href="get-characteristics.html">Get Characteristics</a> - get all characteristics of an advertised service from a BLE Device.</p>
 
-<h3>Advanced</h3>
+<h3>Intermediate</h3>
```
Comment (jyasskin) [inline web-bluetooth/index.html]: What if this were titled something about "Combining multiple operations"? From https://github.com/GoogleChrome/samples/pull/360#issuecomment-223963940, it seems like that's the unifying trait that distinguishes samples in this section from the "Beginner" samples.

Comment (beaufortfrancois) [inline web-bluetooth/index.html]: I like this!

**Pull request merged**